### ISSUE TITLE
SQLITE_NOTADB: Some Linux users cannot use SQLCIPHER

### ISSUE THREAD

**Issue by MrPrimate** [MrPrimate/ddb-adventure-muncher#8]
Some linux users (notably fast moving distros like Arch and manjaro - @BenjaminPrice  ) have reported the following error.

```

[2021-10-17 18:58:08.766] [error] Error: SQLITE_NOTADB: file is not a database
--> in Database#each('\n' +
  'SELECT ID as id, CobaltID as cobaltId, ParentID as parentId, Slug as slug, Title as title, RenderedHTML as html\n' +
  'FROM Content\n', [Function: rowGenerate], [AsyncFunction: collectionFinished])
    at Database.<anonymous> (/tmp/.mount_ddb-aduIneDx/resources/app.asar/munch/book.js:1506:8)
    at Object.getData (/tmp/.mount_ddb-aduIneDx/resources/app.asar/munch/book.js:1497:6)
    at /tmp/.mount_ddb-aduIneDx/resources/app.asar/main.js:232:14
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    
  ```
  
I believe this is due to https://github.com/journeyapps/node-sqlcipher/issues/81, https://github.com/sqlcipher/sqlcipher/issues/385 as GTK3 now bundles SQLite (https://gitlab.gnome.org/GNOME/tracker/-/issues/289) and this is used instead of the sqlcipher.

**Comment by HLennart** [comment]
I have the problem on arch, installing [sqlcipher](https://archlinux.org/packages/community/x86_64/sqlcipher/) and running the appimage via 
> ```LD_PRELOAD=/usr/lib/libsqlcipher.so ./ddb-adventure-muncher-0.4.5-linux-x86_64.AppImage```

 seems to solve it

**Comment by veedubb65** [comment]
> I have the problem on arch, installing [sqlcipher](https://archlinux.org/packages/community/x86_64/sqlcipher/) and running the appimage via
> 
> > `LD_PRELOAD=/usr/lib/libsqlcipher.so ./ddb-adventure-muncher-0.4.5-linux-x86_64.AppImage`
> 
> seems to solve it

Nothing but issues on Ubuntu until I installed sqlcipher. Thank you!!!!!

**Comment by ArktinenSieni** [comment]
> I have the problem on arch, installing [sqlcipher](https://archlinux.org/packages/community/x86_64/sqlcipher/) and running the appimage via
> 
> > `LD_PRELOAD=/usr/lib/libsqlcipher.so ./ddb-adventure-muncher-0.4.5-linux-x86_64.AppImage`
> 
> seems to solve it

Worked like a charm.

**Comment by Mojo9** [comment]
im not sure if I do something wrong but the workaround did not solve the error for me. im running on manjaro

**Comment by MrPrimate** [comment]
This should be resolved in v0.6.0
